script/download-worker-release.cmd

```diff
@@ -0,0 +1,14 @@
+@echo off
+
+setlocal
+
+set OutputDir=%1
+cd %OutputDir%
+
+echo "Download oldest backwards compatible worker release- 0.9.0 for current version 0.10.0."
```

Comment:
Let's remove the current version since we don't want to update this file for every release.

Reply:
Currently this might have to be updated for the oldest compatible version if there are further incompatibilities, but once we hit 1.0 this file would not have to change, correct?